db/fixtures/customization_templates.yml

```diff
@@ -274,3 +274,10 @@
     />\n</unattend>"
   :type: CustomizationTemplateSysprep
   :system: true
+
+- :name: Basic root pass template
+  :description: This template takes use of rootpassword defined in the UI
+  :script: "#cloud-config\nchpasswd:\n  list: |\n    root:<%= MiqPassword.decrypt(evm[:root_password]) %>\n  expire: False"
```

Comment:
Didn't work, there seems to be a bug of some sort, i think the initializers aren't loading like you suggested.